src/Simplex/Chat.hs

```diff
@@ -5360,20 +5420,20 @@ processAgentMessageConn vr user@User {userId} corrId agentConnId agentMessage =
           | senderRole < GRAdmin -> messageError $ "x.msg.del: message not found, message of another member with insufficient member permissions, " <> tshow e
           | otherwise -> withStore' $ \db -> createCIModeration db gInfo m msgMemberId sharedMsgId msgId brokerTs
       where
-        deleteMsg :: MsgDirectionI d => GroupMember -> ChatItem 'CTGroup d -> CM ()
-        deleteMsg mem ci = case sndMemberId_ of
+        deleteMsg :: GroupMember -> CChatItem 'CTGroup -> CM ()
+        deleteMsg mem cci = case sndMemberId_ of
```

Comment:
no need to change name when type changes, it only creates noise

Reply:
we used `cci` in the context when both CChatItem and ChatItem were needed, to differentiate, it doesn't mean we have to use it every time it's CChatItem